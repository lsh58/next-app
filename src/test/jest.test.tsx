import { render, screen } from "@testing-library/react";
import Jest from "@/pages/jest";

describe("<Jest />", () => {
  it("renders a heading", () => {
    const { container } = render(<Jest />);

    const jest = screen.getByText("JestTest");

    expect(jest).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
