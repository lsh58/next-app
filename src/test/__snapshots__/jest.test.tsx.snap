// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Jest /> renders a heading 1`] = `
<div>
  <div>
    JestTest
  </div>
</div>
`;
